Dockerfile

```diff
@@ -1,19 +1,17 @@
-ARG OS_VERSION=bionic
-FROM ubuntu:${OS_VERSION}
+FROM centos:7.8.2003
 
-ARG DEBIAN_FRONTEND=noninteractive
-ARG GIT_URL="https://github.com/ceph/ceph"
-ARG GIT_REF="master"
+ARG GIT_URL="https://github.com/uccross/skyhookdm-ceph"
+ARG GIT_REF="skyhook-luminous"
 ARG EXTRA_PKGS=""
 
-RUN apt-get update && \
-    apt-get install -y git gnupg2 ccache && \
+ADD . /
+
+RUN ./install-preq.sh && \
     git clone --branch $GIT_REF --depth 1 $GIT_URL ceph && \
     cd ceph && \
     ./install-deps.sh && \
     sh -c 'if [ -n "$EXTRA_PKGS" ]; then apt-get install -y "$EXTRA_PKGS"; fi' && \
```

Comment:
for centos, apt-get cannot run. So we can remove this line

Reply:
oh yeah. missed it.